zipline/sources/requests_csv.py

```diff
@@ -138,8 +139,7 @@ def mask_requests_args(url, validating=False, params_checker=None, **kwargs):
     return request_pair(requests_kwargs, url)
 
 
-class PandasCSV(object):
-    __metaclass__ = ABCMeta
```

Comment:
Thinking about the failure mode here, I guess people could instantiate it on py3, nice.